runtime/bindings/python/src/pyopenvino/core/offline_transformations.cpp

```diff
@@ -81,4 +84,26 @@ void regmodule_offline_transformations(py::module m) {
         },
         py::arg("function"),
         py::arg("param_res_names"));
+
+    m_offline_transformations.def(
+        "compress_model_transformation",
+        [](std::shared_ptr<ov::Function> function) {
+            ov::pass::Manager manager;
+            manager.register_pass<ov::pass::MarkPrecisionSensitiveSubgraphs>();
+            manager.register_pass<ov::pass::CompressFloatConstants>();
+            manager.run_passes(function);
+        },
+        py::arg("function"));
+
+    // todo: remove as serialize as part of passManager api will be merged
+    m_offline_transformations.def(
+        "serialize",
+        [](std::shared_ptr<ov::Function> function, const std::string& path_to_xml, const std::string& path_to_bin) {
+            ov::pass::Manager manager;
+            manager.register_pass<ov::pass::Serialize>(path_to_xml, path_to_bin);
```

Comment:
Original C++ API accepts 4-th argument "Version", which can be "unspecified, ir_v10, ir_v11". Can it be exposed as well to Python users? We just hit into this issue with @artyomtugaryov